news/2025/2025-04-15-the-followpoint-azer-the-statistics-breaking-magician.md

```diff
@@ -0,0 +1,305 @@
+
+[OWC 2017 vs Germany: Oriental Blossom](https://www.youtube.com/watch?v=pll8GQvpUUk)
+
+During this year of OWC, I was really making a name for myself being one of the most prominent players of tech/gimmick picks. The week prior, I [SS'd sewing machine](https://youtu.be/Y7DsOtMKfaI?si=z8uV1yWkSg1iseW0&t=3931) against Austria and proved that these types of maps were really dangerous to play against us. A week later, it was picked against germany and despite that team being astronomically favored to win against us (this is the only map we won lol), it felt great to do it again and showing that it wasn't a fluke the first time!
+
+[OWC 2018 vs Australia: First pick strategy](https://youtu.be/xGXobUU6wdY?si=yQjhdUfAI4AxqPUi&t=592) (This video has corrupt audio near the end of the first map, sorry!)
+
+This one requires some context, but I felt like I had to include this moment as a captain and not as a player. We had a really tough matchup against team Australia, who this year (and ever since) was arguably the strongest speed/DT roster of the entire tournament. We knew for SURE that they would win the 2 remaining picks of that type we could not ban out, so instead of letting that happen, I came up with the idea of grabbing the first pick and preying on tournament nerves being at their highest on match start. This was 100% lost in normal conditions and everyone in my team knew this. We essentially stole one of their best picks and forced them to play it at the worst possible conditions, while the rest of our team just had fun **assuming that the pick would be lost**. As it turns out, when you play with no nerves with the expectation of losing, you end up playing pretty well, so we somehow won that pick due to this strategy. It was extremely crucial in the match and took us to tiebreaker, which we also ended up winning. This is definitely still the best tournament strategy I came up with to this day!
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Did we kick a kid? :tf:
```

Comment:
the fa page and the united clip feels appropriate to link here but it's not the entire context... idk what would be the best thing to link for the specific mishearing

```suggestion
[Did we kick a kid?](https://osu.ppy.sh/beatmaps/artists/200) :tf:
```

Reply:
FA page works imo